index.html

```diff
@@ -5,6 +5,8 @@
     <meta name="robots" content="notranslate" />
     <meta name="googlebot" content="notranslate" />
     <meta charset="utf-8" />
+    <link rel="stylesheet" href="src/v3/global/global.css" />
```

Comment:
Hi @Unviray, thanks for preparing this. Do we really need these two imports here? Actually, the `src/v3/**` folder no longer valid when the code gets compiled. So this won’t work. Thanks.

Reply:
This will load all CSS variable colors.

However, I can hardcode the default color (--accent-*) for the `[data-theme='blue-light']` theme in a <noscript> style block.

Or is there a better approach to achieve this?

Reply:
I think we should go with what you described: using only what we need for this `noscript` without importing all unnecessary styles.